src/circuits/gates.jl

```diff
@@ -73,14 +73,15 @@ gate(::GateName"H") = [
 ]
 
 # Rϕ with ϕ = π/2
-gate(::GateName"Phase") = [
+gate(::GateName"Phase"; ϕ::Number = π/2) = [
   1 0
-  0 im
+  0 exp(im*ϕ)
 ]
 
-gate(::GateName"P") = gate("Phase")
+gate(::GateName"P"; kwargs...)     = gate("Phase"; kwargs...)
+gate(::GateName"PHASE"; kwargs...) = gate("Phase"; kwargs...)
 
-gate(::GateName"S") = gate("Phase")
+gate(::GateName"S"; kwargs...) = gate("Phase"; kwargs...)
```

Comment:
Should we define this as:
```julia
gate(::GateName"S") = gate("Phase"; ϕ=π/2)
```
to match the Wikipedia definition? (https://en.wikipedia.org/wiki/Quantum_logic_gate#Phase_shift_gates)

Reply:
Good catch